resources/keb/values.yaml

```diff
@@ -644,3 +478,398 @@ providersConfiguration:
       ap-ae-1:
         displayName: "ap-ae-1"
         zones: [ "a", "b" ]
+
+# Defines which machine type families are available in which regions (and optionally, zones).
+# Restricts provisioning of listed machine types to the specified regions/zones only.
+# If a machine type is not listed, it is considered available in all regions.
+regionsSupportingMachine: |-
+
+# Defines the default KymaCR template.
+runtimeConfiguration: |-
+  default: |-
+    kyma-template: |-
+      apiVersion: operator.kyma-project.io/v1beta2
+      kind: Kyma
+      metadata:
+        labels:
+          "operator.kyma-project.io/managed-by": "lifecycle-manager"
+        name: tbd
+        namespace: kcp-system
+      spec:
+        channel: fast
+        modules: []
+    additional-components: []
+
+# Restricts where the sap-converged-cloud plan is available. If a region is not listed, the plan is not offered for that platform region.
+sapConvergedCloudPlanRegionMappings: |-
+  cf-eu20-staging:
+    - eu-de-1
+    - eu-de-2
+  cf-eu21-staging:
+    - eu-de-3
+    - eu-de-4
+
+# Contains DNS provider configuration for SKR clusters.
+skrDNSProvidersValues: |-
+  providers: []
+
+# Contains the default OIDC configuration for SKR clusters.
+skrOIDCDefaultValues: |-
+  clientID: "9bd05ed7-a930-44e6-8c79-e6defeb7dec9"
+  groupsClaim: "groups"
+  groupsPrefix: "-"
+  issuerURL: "https://kymatest.accounts400.ondemand.com"
+  signingAlgs: [ "RS256" ]
+  usernameClaim: "sub"
+  usernamePrefix: "-"
+
+stepTimeouts:
+  # Maximum time to wait for a runtime resource to be created before considering the step as failed (e.g., 60m = 60 minutes).
```

Comment:
```suggestion
  # Maximum time to wait for a runtime resource to be created before considering the step as failed, for example, 60m = 60 minutes.
```

Reply:
Do we need ` = 60 minutes`? Is there scope for confusion if you only use `60m`? The same applies to lines 529 and 532.

Reply:
I will remove it.